README.md

```diff
@@ -48,7 +48,7 @@ config :nadia,
 ```
 
 And then, in `mix.exs`, list `:nadia` as an application inside `application/0`:
-
+(you can omit this if your elixir is >= 1.4)
```

Comment:
I think we could use more explanation here, e.g.,
"you can leave the `applications` key empty to use [application inference](https://elixir-lang.org/blog/2017/01/05/elixir-v1-4-0-released/) if your elixir is >= 1.4"